src/Appwrite/Services/Functions.cs

```diff
@@ -510,15 +562,18 @@ public Task<Models.ExecutionList> ListExecutions(string functionId, List<string>
         /// function execution process will start asynchronously.
         /// </para>
         /// </summary>
-        public Task<Models.Execution> CreateExecution(string functionId, string? data = null, bool? xasync = null)
+        public Task<Models.Execution> CreateExecution(string functionId, string? body = null, bool? xasync = null, string? xpath = null, string? method = null, object? headers = null)
```

Comment:
`object? headers` cannot be type casted implicitly from `object?` to `Dictionary<string, string>?`

We'll have to mention the right type when declaring